core/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/AwarenessAllocationDecider.java

```diff
@@ -19,21 +19,20 @@
 
 package org.elasticsearch.cluster.routing.allocation.decider;
 
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
Your IDE seems to be non-consistent about putting these places. In some files you put them above the ES imports, in others, below (see AllocationDeciders.java). I think we decided originally to be consistent with putting them below the ES imports?